tests/appengine/util/ArrayUtilTest.php

```diff
@@ -18,7 +18,7 @@
 
 use google\appengine\util\ArrayUtil;
 
-class ArrayUtilTest extends \PHPUnit_Framework_TestCase {
+class ArrayUtilTest extends \PHPUnit\Framework\TestCase {
```

Comment:
nit: import the `TesCase` class

```suggestion
use \PHPUnit\Framework\TestCase;

class ArrayUtilTest extends TestCase {
```

Reply:
Updated this case for all unit tests, thanks!